.gitignore

```diff
@@ -12,3 +12,4 @@ yarn.lock
 # IDEs
 .idea/
 .vs/
+.husky/
```

Comment:
This isn't an IDE and is required, so it shouldn't be added to a `.gitignore` file.

Reply:
Still needs resolving.